src/Files.App/UserControls/NavigationToolbar.xaml

```diff
@@ -384,10 +384,15 @@
 						<controls:BreadcrumbBar.ItemTemplate>
 							<DataTemplate x:DataType="dataitems:PathBoxItem">
 								<controls:BreadcrumbBarItem
+									AllowDrop="True"
 									AutomationProperties.AccessibilityView="Content"
 									AutomationProperties.Name="{x:Bind Title, Mode=OneWay}"
 									ChevronToolTip="{helpers:ResourceString Name=BreadcrumbBarChevronButtonToolTip}"
-									Content="{x:Bind Title, Mode=OneWay}" />
+									Content="{x:Bind Title, Mode=OneWay}"
+									DataContext="{x:Bind}"
+									DragLeave="PathBoxItem_DragLeave"
+									DragOver="PathBoxItem_DragOver"
+									Drop="PathBoxItem_Drop" />
```

Comment:
Please use a more proper name: BreadcrumbBarItem_Drop, etc. PathBoxItem has been inappropriate.

Reply:
Is it the same event that works with the existing Breadcrumb box, as well as the Omnibar one?  If so, **BreadcrumbItem_Drop/DragOver/DragLeave** is approriate.  Not BreadcrumbBar.

If these are separate events only for the Omnibar, then something like **OmnibarBreadcrumbItem_Drop/DragOver/DragLeave** would be more accurate.